product/models.py

```diff
@@ -0,0 +1,136 @@
+from django.db      import models
+
+class Category(models.Model):
+    category        = models.CharField(max_length = 50)
```

Comment:
```suggestion
    name = models.CharField(max_length = 50)
```
카테고리의 이름이란 뜻이죠?
나중에 객체에 접근 할 때, category.category 속성에 접근하는 것보다 category.name에 접근 하는 것이 보다 자연스럽습니다. 
따라서 아래 클래스들에 있는 클래스 이름과 그 이름이 같은 속성들은 모두 `name`으로 변경하시는 것을 추천드립니다.